linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_mac_modification_usr_share/oval/shared.xml

```diff
@@ -22,6 +30,11 @@
     </criteria>
   </definition>
 
+{{%- if 'sle15' in product -%}}
+  {{{ oval_test_package_installed(package="selinux-policy-devel", evr=EVR, test_id="check_selinux_policy_devel_installed") }}}
```

Comment:
The `EVR` variable isn't defined. You probably wanted to set it to None or you can remove the ` evr=EVR,` completely because it's a parameter with a default value (see the definition of the `oval_test_package_installed` macro).